components/footer.js

```diff
@@ -0,0 +1,34 @@
+export default function Footer() {
+    return (
+        <div className="footer">
+            <svg id="footer-svg" xmlns="http://www.w3.org/2000/svg" viewBox="0 100 1440 200">
+                <path
+                    fill="#171717"
+                    fillOpacity="1"
+                    d="M0,288L40,266.7C80,245,160,203,240,176C320,149,400,139,480,138.7C560,139,640,149,720,165.3C800,181,880,203,960,197.3C1040,192,1120,160,1200,149.3C1280,139,1360,149,1400,154.7L1440,160L1440,320L1400,320C1360,320,1280,320,1200,320C1120,320,1040,320,960,320C880,320,800,320,720,320C640,320,560,320,480,320C400,320,320,320,240,320C160,320,80,320,40,320L0,320Z"
+                />
+            </svg>
+            <div
+                className="
+                2xl:text-2xl xl:text-xl lg:text-lg md:text-base sm:text-sm text-xs
+                2xl:gap-20 xl:gap-10 lg:gap-10 md:gap-5 gap-2
+                md:h-20 sm:h-60 h-40 bg-black
```

Comment:
this doesn't look linted 🤔

Reply:
oh gosh and also you don't need 21 classes. i achieved it with 8.

Reply:
Lol first time using tailwind CSS my bad - what exactly does linted mean?

Reply:
![image](https://user-images.githubusercontent.com/20258660/113202853-4a2a2800-9239-11eb-8347-93380e91888c.png)
Can't seem to reproduce the issue - what exactly did you do to get there?

Reply:
I'm also not able to reproduce this issue.

Reply:
most likely a smaller viewport width.

Reply:
What width exactly? I'm going to lint and commit for now